samples/sample_encode/src/sample_encode.cpp

```diff
@@ -1119,6 +1144,14 @@ mfxStatus ParseInputString(msdk_char* strInput[], mfxU8 nArgNum, sInputParams* p
         return MFX_ERR_UNSUPPORTED;
     }
 
+    if (pParams->nRateControlMethod == MFX_RATECONTROL_AVBR)
+    {
+        if (pParams->Accuracy > 100)
+        {
+            pParams->Accuracy = 100;
```

Comment:
Shouldn't it be warned?

Reply:
I think the reply "Added prompt here" has lead in confusion here. Sorry for that. I mean I have added prompt info in source codes. Thanks!